source/Octopus.Tentacle.Client/EventDriven/CommandContext.cs

```diff
@@ -0,0 +1,21 @@
+using Octopus.Tentacle.Client.Scripts;
+using Octopus.Tentacle.Contracts;
+
+namespace Octopus.Tentacle.Client.EventDriven
+{
+    public class CommandContext
```

Comment:
nit: consider documenting what this class is for.

Reply:
Good idea. Done